lib/msal-common/src/utils/Constants.ts

```diff
@@ -401,3 +401,21 @@ export enum JsonTypes {
     Jwt = "JWT",
     Jwk = "JWK"
 }
+
+export enum SilentTokenRetrievalStrategy {
```

Comment:
This should be exported from the `index.ts` file for msal-common, and then rexported in msal-browser's `index.ts`.